front/app/components/admin/Table/Td.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+// components
+import { Box, BoxProps } from '@citizenlab/cl2-component-library';
+
+interface Props extends BoxProps {
+  colSpan?: `${number}`;
+}
+
+const Cell = ({ children, colSpan, ...otherProps }: Props) => (
+  <Box as="td" p="12px" colSpan={colSpan as any} {...otherProps}>
```

Comment:
Why is `as any` needed here? You can use a styled box or just pass this in a `style` object for convenience?